woocommerce/woocommerce
# [WIP]: Add/migrate react hook form

### All Submissions:






### Changes proposed in this Pull Request:



Closes #36102  .






### How to test the changes in this Pull Request:



1. Open the new product form
2. Update the name, categories, feature, summary and description fields
3. Check that no unnecessary re-renders occur (you can check it by enabling this checkbox on React Dev Tools profiler tab)
<img width="453" alt="image" src="https://user-images.githubusercontent.com/13437655/215161876-2b542720-0bdb-4595-866d-77ae6aaff043.png">




### Other information:




### FOR PR REVIEWER ONLY:
